templates/zerver/help/send-a-private-message.md

```diff
@@ -4,25 +4,30 @@ Zulip allows users to send private messages to each other.
 
 ## There are two ways to send a Private Message:
 
-### Way 1:
+### Method 1:
```

Comment:
Can you title this with a brief description of the method instead of just saying "method 1"? (FYI, this involves using the compose box so you could that term)